test/any/client/api_matrix-test.es6.js

```diff
@@ -0,0 +1,134 @@
+  describe('toMatrix()', function() {
+    let apiImporter = org.chromium.apis.web.ApiImporter.create();
+    let apiMatrix = org.chromium.apis.web.ApiMatrix.create({
+      browserAPIs: apiImporter.browserAPIs,
+    });
+    apiImporter.import('Chrome', '55', 'Window', '10', {
+      'Array': ['find'],
+      'Audio': ['stop'],
+    });
+    apiImporter.import('Edge', '14', 'Window', '10', {
+      'Array': ['find'],
+      'Audio': ['play'],
+    });
+    apiImporter.import('Safari', '10', 'OSX', '601', {
+      'ApplePay': ['about'],
+      'Audio': ['play', 'stop'],
+      'Array': ['find'],
+    });
+    it(`contains correct interface and API information, when all browsers
+    are selected`, function(done) {
+      apiMatrix.toMatrix([
+        'Chrome_55_Window_10',
+        'Edge_14_Window_10',
+        'Safari_10_OSX_601',
+      ]).then((interfaceMap) => {
+        expect(interfaceMap.ApplePay.about).toEqual({
+          Safari_10_OSX_601: true,
+        });
+        expect(interfaceMap.Array.find).toEqual({
+          Chrome_55_Window_10: true,
+          Edge_14_Window_10: true,
+          Safari_10_OSX_601: true,
+        });
+        expect(interfaceMap.Audio.play).toEqual({
+          Edge_14_Window_10: true,
+          Safari_10_OSX_601: true,
+        });
+        expect(interfaceMap.Audio.stop).toEqual({
+          Chrome_55_Window_10: true,
+          Safari_10_OSX_601: true,
+        });
+        done();
+      });
+    });
+    it(`contains correct interface and API information, when part of
+    keys are selected.`, function(done) {
+      apiMatrix.toMatrix([
+        'Chrome_55_Window_10',
+        'Edge_14_Window_10',
+      ]).then((interfaceMap) => {
+        expect(interfaceMap.Array.find).toEqual({
+          Chrome_55_Window_10: true,
+          Edge_14_Window_10: true,
+        });
+        expect(interfaceMap.Audio.play).toEqual({
+          Edge_14_Window_10: true,
+        });
+        expect(interfaceMap.Audio.stop).toEqual({
+          Chrome_55_Window_10: true,
+        });
+        done();
+      });
+    });
+  });
+
+  describe('toCSV()', function() {
+    let apiImporter = org.chromium.apis.web.ApiImporter.create();
+    let apiMatrix = org.chromium.apis.web.ApiMatrix.create({
+      browserAPIs: apiImporter.browserAPIs,
+    });
+    apiImporter.import('Chrome', '55', 'Window', '10', {
+      'Array': ['find'],
+      'Audio': ['stop'],
+    });
+    apiImporter.import('Edge', '14', 'Window', '10', {
+      'Array': ['find'],
+      'Audio': ['play'],
+    });
+    apiImporter.import('Safari', '10', 'OSX', '601', {
+      'ApplePay': ['about'],
+      'Audio': ['play', 'stop'],
+      'Array': ['find'],
+    });
+    it(`produces correct csv string when all browsers are selected`,
+    function(done) {
+      apiMatrix.toCSV([
+        'Chrome_55_Window_10',
+        'Edge_14_Window_10',
+        'Safari_10_OSX_601',
+      ]).then((csvStr) => {
+        expect(csvStr.split('\n').sort()).toEqual(
```

Comment:
Add sort in every test. Sort is added so that the ordering of row does not affect to the result.